feat: update schemas and submodule for v0.3.2

yanCode opened the pull request:
This issue is detected after I  installed a new vs code extension called [Even Better Toml](https://marketplace.cursorapi.com/items?itemName=tamasfe.even-better-toml)
Recently there are a few commits in https://github.com/sxyazi/yazi, (including this one https://github.com/sxyazi/yazi/commit/92112de1c05c288c631f669523fc40fd4eb709d2) 
adding some new properties like `confirm` ,`title_format` and `image_delay`, which should be sync up to validation schema as well.

This PR has passed this  validation `npm run lint `

sxyazi reviewed (approved):
LGTM, thanks!

@uncenter Shall we merge this PR?

uncenter reviewed (changes requested):
Please see how I have updated to Yazi updates in the past. The submodule needs to be updated to the latest released commit / tag, and the commit message (and PR title) should reflect the changes - this would be a `feat: update schemas and submodule for v0.3.2`.

uncenter commented on schemas/yazi.json:
You've missed the new `preview.wrap` option: https://github.com/sxyazi/yazi/commit/cd7209c040a706bd826b20f47d4accdcf0c5a762#diff-62c176c03cb77cb11e287cf9fe420fe603abd55296017d7323328e37eb056818.

yanCode replied:
```
"wrap": {"enum": ["yes", "no"]},
```
now `preview.wrap` has been added, which support 2 options, `yes` or `no`.

uncenter commented on schemas/yazi.json:
```diff
@@ -223,6 +225,42 @@
 				}
 			}
 		},
+		"confirm": {
+			"type": "object",
+			"properties": {
+				"trash_title": { "type": "string" },
+				"trash_origin": {
+					 "$ref": "#/$defs/origin"
+				},
+				"trash_offset": { 
+					"$ref": "#/$defs/offset" 
+				},
+				"delete_title": { "type": "string" },
+				"delete_origin": { 
+					"$ref": "#/$defs/origin"
+				},
+				"delete_offset": { 
+					"$ref": "#/$defs/offset" 
+				},
+				"overwrite_title": { "type": "string" },
+				"overwrite_content": { "type": "string" },
+				"overwrite_origin": {
+					"$ref": "#/$defs/origin" 
+				},
+				"overwrite_offset": { 
+					"$ref": "#/$defs/offset"
+			  },
+				"quit_title": { "type": "string" },
+				"quit_content": { "type": "string" },
+				"quit_origin": { 
+					"$ref": "#/$defs/origin" 
+				},
+				"quit_offset": { 
+					"$ref": "#/$defs/offset" 
+				}
+			},
+			"additionalProperties": false
+		},
```
You missed removing these properties from the old `input` section as well: https://github.com/sxyazi/yazi/commit/92112de1c05c288c631f669523fc40fd4eb709d2#diff-86ad26ab85d77e4bd594a4813c46c5292a93e9ddfc9f4c224e3b9d8627635b74.

uncenter replied:
No worries. Thanks!

uncenter commented on schemas/yazi.json:
```diff
@@ -53,6 +54,7 @@
 				"tab_size": { "$ref": "/schemas/types/u8.json" },
 				"max_width": { "$ref": "/schemas/types/u32.json" },
 				"max_height": { "$ref": "/schemas/types/u32.json" },
+				"image_delay": { "$ref": "/schemas/types/u8.json" },
```
Nitpicky but this should go below `cache_dir` and above `image_filter`, as is the order in Yazi's source code and preset theme files.

uncenter replied:
This also is missing the minimum (of 0) and maximum (of 100) contraints.

yanCode replied:
Now i moved `image_filter` down blow `cache_dir ` with new a value
```
	"image_delay": {"type": "number", "minimum": 0, "maximum": 100 },
```
which limits the value from minimum 0 to maximum 100. I totally agree with you that the order should be preserved.

yanCode commented:
now I fixed all the issues you mentioned in your review. 
And I updated both the git commit message & PR title as `feat: update schemas and submodule for v0.3.2`, i will follow this rule if I have further opportunities to submit any new PRs.
Many thanks to your detailed review, which helps me a lot to understand this cool project.

uncenter commented:
There is some weird limitation I am getting about signed commits which means I couldn't merge this PR from the web GitHub. I merged it from the command line, see https://github.com/yazi-rs/schemas/commit/ae3146fb5cc55db1c17394a78be1018f7ca6beed.

uncenter commented:
I would have liked to add you as co-author to that commit but due to more branch protections I can't push to main :/

uncenter commented:
FYI, I've now added better contributing documentation: https://github.com/yazi-rs/schemas/blob/main/CONTRIBUTING.md.
